content/doc/developer/publishing/wiki-page.adoc

```diff
@@ -8,54 +8,37 @@ references:
   title: Jenkins Wiki Exporter
 ---
 
-WARNING: The Jenkins wiki was made 'read-only' in link:https://groups.google.com/d/msg/jenkinsci-dev/lNmas8aBRrI/eL3u7A6qBwAJ[October 2019].
-Plugin documentation should be maintained in the GitHub repository of the plugin.
+WARNING: The Jenkins wiki was made 'read-only' in link:https://groups.google.com/d/msg/jenkinsci-dev/lNmas8aBRrI/eL3u7A6qBwAJ[October 2019]. 
+In September 2021, link:https://www.jenkins.io/blog/2021/09/04/wiki-attacked/[Jenkins wiki was attacked] and as a result, the impacted server was permanently disabled and the site was rendered static.
+Plugin documentation is now maintained in the GitHub repository of the plugin.
 See the link:../documentation[plugin documentation guidelines] for details.
 
 Plugin documentation is available from the link:https://plugins.jenkins.io[Jenkins plugins site] so that potential users can learn about the plugin without installing it.
 
 == Migrating from Wiki to GitHub
 
-Document migration can be done in a semi-automated way using the link:https://pandoc.org[Pandoc] tool for Wiki conversion to Markdown or Asciidoc.
-This guide shows how to do so for a common case when the documentation page is going to be sourced from the README file inside the repository's root.
+All plugin documentation have been converted to markdown and are stored in the link:https://github.com/jenkins-infra/plugins-wiki-docs/[plugins-wiki-docs] repository.
```

Comment:
Singular rather than plural

```suggestion
All plugin documentation has been converted to markdown and is stored in the link:https://github.com/jenkins-infra/plugins-wiki-docs/[plugins-wiki-docs] repository.
```